cli/tests/testdata/npm/deno_run_no_bin_entrypoint_non_existent_subpath.out

```diff
@@ -1,3 +1,3 @@
-error: package '@denotest/esm-basic' did not have a bin property in its package.json
+error: '[WILDCARD]@denotest[WILDCARD]esm-basic[WILDCARD]package.json' did not have a bin property
```

Comment:
I assume this is now displaying the full path?

Reply:
Yeah because it's working off paths now.